Update README.md (#84)
online-judge-tools/template-generator: README.md

### oj-prepare

> The config file for `oj-prepare` can be found at the following paths according to your operating system:

Reviewer: ```suggestion
The config file for `oj-prepare` can be found at the following paths depending on your operating system:
```
